openforcefield/topology/topology.py

```diff
@@ -1660,7 +1660,14 @@ def from_openmm(cls, openmm_topology, unique_molecules=None):
                     break
             if not (match_found):
                 hill_formula = Topology._networkx_to_hill_formula(omm_mol_G)
-                raise ValueError(f'No match found for molecule {hill_formula}')
+                msg = f'No match found for molecule {hill_formula}. '
+                probably_missing_conect = ['C1', 'H1', 'O1', 'N1', 'P1', 'S1', 'F1', 'Cl1', 'Br1']
+                if hill_formula in probably_missing_conect:
+                    msg += 'This would be a very unusual molecule to try and parameterize, ' \
```

Comment:
In case you haven't stumble on this yet, there's an alternative suggested way of splitting strings on multiple lines with parentheses
```python
msg += ('This would be a very unusual molecule to try and parameterize, '
        'and it is likely that the data source it was read from does not ')
```

Reply:
Oh nice, converted to that format.